autodocsumm/__init__.py

```diff
@@ -303,8 +310,14 @@ def add_autosummary(self):
                 indent = '    '
 
                 for (documenter, _) in documenters:
-                    self.add_line(
-                        indent + '~' + documenter.fullname, sourcename)
+                    if relative_ref_paths:
+                        obj_ref_path = documenter.fullname.lstrip(
+                            self.modname + '.')
+                    else:
+                        obj_ref_path = documenter.fullname
```

Comment:
as discussed in https://github.com/Chilipp/autodocsumm/pull/70/files#r852889338 but a bit more clean

```suggestion
                    obj_ref_path = documenter.fullname
                    if relative_ref_paths:
                        modname = self.modname + "."
                        if documenter.fullname.startswith(modname):
                            obj_ref_path = documenter.fullname[len(modname):]
```